proxy/proxy.ts

```diff
@@ -0,0 +1,126 @@
+// @ts-ignore
```

Comment:
Let’s define simple types in `global.d.ts`. It should be a few lines just for API which we use.